### PR TITLE
Support for clean & safe error handling on IE 11

### PR DESCRIPTION
**Pull request by aaron-costello**
Added pad_utils sanitization for clean and safe error handling on browsers that do not encode the path of the URL.

### CODE REVIEW

**Comment by muxator** [comment]
Thanks.

The patch was applied on **1.7.5** and applies cleanly on 1.8 as well.
Rebasing there & force pushing on this PR to keep history auditable.

File: src/templates/pad.html [hunk 1]
```diff
@@ -400,6 +401,7 @@ <h2 data-l10n-id="pad.share.emebdcode"></h2>
 
               var plugins = require('ep_etherpad-lite/static/js/pluginfw/client_plugins');
               var hooks = require('ep_etherpad-lite/static/js/pluginfw/hooks');
+              var padutils = require('../static/js/pad_utils');
```
Comment (muxator) [inline src/templates/pad.html]: Is this require() needed?

Comment (muxator) [reply src/templates/pad.html]: I guess it's not (only the first one seems to be needed).

Comment (aaron-costello) [reply src/templates/pad.html]: Hi @muxator ,

Apologies I didn't notice that was still in there, that 2nd one is a result of testing and is not effective. Only the first 'require' is needed.

**Comment by muxator** [comment]
The PR cannot be rebased on `develop` (where patches should be directed).

I'll manually apply this & close the PR.

**Comment by muxator** [comment]
I think this change broke the frontend code.

This is what my developer tools show when loading a pad in etherpad **1.8.0-beta.1** (I am using Firefox 70):
```
ReferenceError: require is not defined
    <anonymous> https://beta.etherpad.org/p/issue3647:559
```

@aaron-costello, do you see this as well?


I have left a comment in the commit that implements this PR:
https://github.com/ether/etherpad-lite/commit/5879037ddca4ab9a4002adf90fc7ce6c9f82f01b#diff-af61109c6bb3440a9cb12337af45894eR364

**Pull request closed without merging**